scripts/skills/actives/rf_gouge_skill.nut

```diff
@@ -0,0 +1,123 @@
+this.rf_gouge_skill <- ::inherit("scripts/skills/skill", {
+	m = {
+		BleedStacks = 3,
+		SoundsA = [
+			"sounds/combat/cleave_hit_hitpoints_01.wav",
+			"sounds/combat/cleave_hit_hitpoints_02.wav",
+			"sounds/combat/cleave_hit_hitpoints_03.wav",
```

Comment:
I made you add a whole bunch of commas at the end of arrays/tables during the commit 48824bc30d432ec2710efaf219a313aace5e2818
But that rule was very quickly abolished so the commas should be removed again